CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing to CodeIgniter4
+
+CodeIgniter is a community driven project and accepts contributions of
+code and documentation from the community.
+
+If you'd like to contribute, please read the [Contributing to CodeIgniter](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/README.md)
+in the [main repository](https://github.com/codeigniter4/CodeIgniter4).
+
+If you are going to contribute to this repository, please report bugs or send PRs
```

Comment:
[report bugs](https://github.com/codeigniter4/shield/issues/new?assignees=&labels=bug&template=bug_report.yml&title=Bug%3A+) or [send PRs](https://github.com/codeigniter4/shield/compare)

Reply:
Edited to turn into a suggestion (it was displaying as links in comment).

Reply:
Done.